src/System/HardwareRotaryEncoder.h

```diff
@@ -55,6 +55,7 @@ class HardwareRotaryEncoder : public HardwareInput {
         pin_size_t _pinB;
         PinStatus _lastA;
         input_data_t _counter = 0;
+        uint32_t _doubleCounter = 0;
```

Comment:
##### Copied from commit page:

Won't this underflow right away if you rotate the rotary encoder backwards when the application first starts?  Then you'll get _counter = (2^32-2) / 2

This should be a signed int like `input_data_t` (below is the typedef for `input_data_t`)
```c++
typedef int32_t input_data_t;
```

Reply:
Just coded up an example of this [here](https://onlinegdb.com/eWRyS3QWN)